Sources/SwiftBSON/BSONTimestamp.swift

```diff
@@ -91,6 +91,20 @@ public struct BSONTimestamp: BSONValue, Equatable, Hashable {
         ]
     }
 
+    /// Compares two `BSONTimestamp` instances as outlined by the Comparable protocol
```

Comment:
```suggestion
    /// Compares two `BSONTimestamp` instances as outlined by the `Comparable` protocol.
```

(generally we end sentences in docstrings with periods)